BraveSharedTests/CertificatePinningTest.swift

```diff
@@ -109,7 +109,7 @@ class CertificatePinningTest: XCTestCase {
 
         let host = "unit-test.brave.com"
```

Comment:
I'm not sure what this one is testing because that hostname doesn't resolve.

If you want to test a correctly-pinned site, you can use `pcdn.brave.com`.

If you want to test an incorrectly-pinned site, you can use `ssl-pinning.someblog.org` (also controlled by us).

Reply:
It tests that our certificate-pinning can work on self-signed certificates if we allow it via the options enum.
So it is essentially just testing that anchoring works (without host validation, as a separate test validates the host)